lib/utils/initialAppConfig.ts

```diff
@@ -0,0 +1,101 @@
+import { MLSpaceConfig } from "./configTypes";
+
+
+export function generateAppConfig (mlspaceConfig: MLSpaceConfig) {
+    return {
+        'versionId': {'N': '0'}, 
+        'changedBy': {'S': 'InitialConfig'}, 
+        'configScope': {'S': 'global'}, 
+        'changeReason': {'S': 'Initial deployment default config'},
+        'configuration': {'M': {
+            'EnabledServices': {'M': {
+                'cluster': {'BOOL': 'True'}, 
+                'endpoint': {'BOOL': 'True'}, 
+                'labeling-job': {'BOOL': mlspaceConfig.ENABLE_GROUNDTRUTH ? 'True' : 'False'}, 
+                'transform-job': {'BOOL': 'True'}, 
+                'notebook-instance': {'BOOL': 'True'}, 
+                'real-time-translate': {'BOOL': mlspaceConfig.ENABLE_TRANSLATE ? 'True' : 'False'}, 
+                'training-job': {'BOOL': 'True'}, 
+                'batch-translate-job': {'BOOL': mlspaceConfig.ENABLE_TRANSLATE ? 'True' : 'False'}, 
+                'model': {'BOOL': 'True'}, 
+                'hpo-job': {'BOOL': 'True'}, 
+                'endpoint-config': {'BOOL': 'True'}
+            }}, 
+            'ProjectCreation': {'M': {
+                'AdminOnly': {'BOOL': 'True'}, 
```

Comment:
This should be false right? Currently we allow everyone to create projects by default.

Reply:
Good point, will update